code/framework/encryption/src/main/java/io/cattle/platform/framework/encryption/EncryptionUtils.java

```diff
@@ -0,0 +1,52 @@
+package io.cattle.platform.framework.encryption;
+
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+public class EncryptionUtils {
+    private static String fileName = "keys.json";
+
+    public static boolean isEqual(byte[] a, byte[] b) {
+        if (a.length != b.length) {
```

Comment:
You think it would be worth putting not null checks for a/b to avoid NPEs? As there is a chance "transform" attribute is put on the field allowing null values.